src/bounds.jl

```diff
@@ -1,22 +1,104 @@
 """
-    initialize_tight_bounds(m::PODNonlinearModel)
+    init_tight_bound(m::PODNonlinearModel)
```

Comment:
Later what operations?

Reply:
This function sets up the placeholders for l_var_tight and u_var_tight. These vectors are then used for other operations such as tightening, propagation, etc. I've re-worded this comment in the most recent commit.